### DISCUSSION TITLE
GitHub account suspended

### DISCUSSION THREAD

**Discussion by ghost** [community/community#135391] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

GitHub Education

### Body

My username is ethioCodingRooms. My account was suspended while participating in GitHub's 10th anniversary celebrations. What can I will do? Please help me.

Best regards,
Asres Gamu Yelia

**Comment by Arnav7x** [comment]
Firstly, I would recommend checking the terms and services, maybe you accidently violated a guideline.

Otherwise:

Contact GitHub Support: Reach out to GitHub Support to inquire about the suspension and request reactivation. You can do this by submitting a ticket through their [Support page](https://support.github.com/contact). Provide as much detail as possible about your situation, including any relevant account information and the context of your participation in the anniversary celebrations.

Check Your Email: GitHub usually sends an email explaining the reason for suspension. Check your registered email inbox and spam folder for any communication from GitHub. If you find an email, follow any instructions provided.

Appeal the Suspension: If you believe your account was suspended in error, you can appeal by providing evidence and a clear explanation of your activities that led to the suspension. Be respectful and concise in your communication with GitHub Support.

Avoid Creating a New Account: It's generally not advisable to create a new account to bypass a suspension, as this could lead to further issues. Focus on resolving the matter with your existing account through the official channels.

**Comment by ghost** [comment]
Thank more they not answred me? I emailed to them but not reponsed me.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @asresgamuyahoo, 

Thank you for your patience. The community is unable to assist with nor escalate your support ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note: The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
